### ISSUE TITLE
ModuleNotFoundError: No module named 'tabulate'

### ISSUE THREAD

**Issue by sebma** [shidenko97/pypisearch#5]
Hi,
I just installed `pypisearch` : 
```bash
$ sudo -H $(which pip3.8) install pypisearch
[sudo] password for sebastien:              
Collecting pypisearch
  Downloading pypisearch-1.2.0-py3-none-any.whl (4.9 kB)
Installing collected packages: pypisearch
Successfully installed pypisearch-1.2.0
```

Here's what I get when I use it : 
```python
$ python -m pypisearch speedtest
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.8/dist-packages/pypisearch/__main__.py", line 1, in <module>
    from pypisearch.main import main
  File "/usr/local/lib/python3.8/dist-packages/pypisearch/main.py", line 3, in <module>
    from pypisearch.search import Search
  File "/usr/local/lib/python3.8/dist-packages/pypisearch/search.py", line 3, in <module>
    import tabulate
ModuleNotFoundError: No module named 'tabulate'
```

Can you please help ?

**Comment by shidenko97** [comment]
Hi! Good job man, my fail, sorry, Few moments I will fix it, thanks for ur support!

**Comment by shidenko97** [comment]
@sebma 
Ok, it's done
You should update package up to 1.3.0 version and everything will work)

**Comment by sebma** [comment]
@shidenko97 Man, that was fast !

You fixed this issue via [commit 2f7cb2c](../commit/2f7cb2c) right ?

It seemed the `requirements.txt` file was ignored but now the installation installed the dependencies : 
```
$ sudo -H $(which pip3.8) install pypisearch
[sudo] password for sebastien:              
Collecting pypisearch
  Downloading pypisearch-1.3.0-py3-none-any.whl (5.0 kB)
Collecting chardet==4.0.0
  Downloading chardet-4.0.0-py2.py3-none-any.whl (178 kB)
     |████████████████████████████████| 178 kB 6.9 MB/s 
Collecting idna==2.10
  Downloading idna-2.10-py2.py3-none-any.whl (58 kB)
     |████████████████████████████████| 58 kB 3.5 MB/s 
Collecting certifi==2020.12.5
  Downloading certifi-2020.12.5-py2.py3-none-any.whl (147 kB)
     |████████████████████████████████| 147 kB 11.9 MB/s 
Collecting requests==2.25.1
  Downloading requests-2.25.1-py2.py3-none-any.whl (61 kB)
     |████████████████████████████████| 61 kB 4.3 MB/s 
Requirement already satisfied: tabulate==0.8.9 in /usr/local/lib/python3.8/dist-packages (from pypisearch) (0.8.9)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/lib/python3/dist-packages (from requests==2.25.1->pypisearch) (1.25.8)
Installing collected packages: idna, chardet, certifi, requests, pypisearch
  Attempting uninstall: idna
    Found existing installation: idna 2.8
    Uninstalling idna-2.8:
      Successfully uninstalled idna-2.8
  Attempting uninstall: chardet
    Found existing installation: chardet 3.0.4
    Uninstalling chardet-3.0.4:
      Successfully uninstalled chardet-3.0.4
  Attempting uninstall: certifi
    Found existing installation: certifi 2019.11.28
    Uninstalling certifi-2019.11.28:
      Successfully uninstalled certifi-2019.11.28
  Attempting uninstall: requests
    Found existing installation: requests 2.22.0
    Uninstalling requests-2.22.0:
      Successfully uninstalled requests-2.22.0
Successfully installed certifi-2020.12.5 chardet-4.0.0 idna-2.10 pypisearch-1.3.0 requests-2.25.1
```

And now it works find, thanks a lot : 
```
$ python -m pypisearch pulsemixer
pulsemixer (1.5.1)        [installed 1.5.0] pulsemixer - CLI and curses mixer for PulseAudio
pulsectl-asyncio (0.1.5)  Asyncio frontend for the pulsectl Python bindings of libpulse
pulsectl (21.3.4)         Python high-level interface and ctypes-based bindings for PulseAudio (libpulse)
```

Congratulations !

**Comment by shidenko97** [comment]
Yes, I've skipped dependencies before(
